turtle.js

```diff
@@ -0,0 +1,138 @@
+// worked on it on Monday Nov 19, 2018 at 930pm
+class Turtle {
+
+    constructor (x, y){
```

Comment:
when defining a function, please be consistent with your spacing. I notice here you use the following:
```js
functionName (param1, param2){
```
while on line `13` you do
```js
functionName(){
```
and on `20` you do
```js
functionName() {
```

I prefer the following spacing:
```js
functionName(param1, param2) {
```

This consistency is important for potential employers looking at your code. It shows attention to detail. Similarly, so does consistently with using (or not using) `;`.  To be inconsistent with either is akin to having typos in your resume. It immediately sends the wrong first impression.

Reply:
Thanks Max.  Changes made.